### ISSUE TITLE
Column description is not visible in contrast theme

### ISSUE THREAD

**Issue by MashaKorax** [dbeaver/dbeaver#35629]
### Description

contrast theme
![dbeaver_I6mpKGGxuQ](https://github.com/user-attachments/assets/18e0b46b-2c64-4a04-afc7-edb7b667c7df)

dark theme
![dbeaver_6kTkctKz8M](https://github.com/user-attachments/assets/ce5370c6-f2f5-45f9-9197-cea44d4489c7)

### DBeaver Version

DBeaver 24.2.1

### Operating System

windows 10
